config/core/300-resources/configuration.yaml

```diff
@@ -462,6 +462,10 @@ spec:
                         enableServiceLinks:
                           description: 'EnableServiceLinks indicates whether information about services should be injected into pod''s environment variables, matching the syntax of Docker links. Optional: Defaults to true.'
                           type: boolean
+                        idleTimeoutSeconds:
+                          description: IdleTimeoutSeconds is the maximum duration in seconds a request will be allowed to stay open while not receiving any bytes from the user's application. If unspecified, a system default will be provided.
```

Comment:
The default is 0, which would be an infinite timeout, right? May want to be a bit more explicit, something like "If unspecified, no idle timeout will be used."

Reply:
Actually that reminds me of something I missed. There is a `revision-timeout-seconds` in the `config-defaults` configmap which allows people to set the default for the first byte timeout. I think we should have the same for idle timeout so that system admin can configure its default if they don't want it to be always 0. Let me look into that.

Reply:
Pushed a new commit https://github.com/knative/serving/pull/11988/commits/813d0ddd64dd66870483639ebe59b4ff3552b479 now it's "If unspecified, a system default will be provided." indeed